packages/date-time/src/components/DatePicker/__snapshots__/DatePicker.test.tsx.snap

```diff
@@ -121,30 +121,24 @@ exports[`DatePicker renders default DatePicker correctly 1`] = `
                   text-overflow: ellipsis;
                   width: 100%;
                 }
-                &:active, &:focus, &:hover {
-                  outline: 0px;
-                }
-                &::-ms-clear {
-                  display: none;
-                }
                 &::placeholder {
-                  -moz-osx-font-smoothing: grayscale;
-                  -webkit-font-smoothing: antialiased;
                   color: #605e5c;
-                  font-family: 'Segoe UI', 'Segoe UI Web (West European)', 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif;
-                  font-size: 14px;
-                  font-weight: 400;
                   opacity: 1;
                 }
                 &:-ms-input-placeholder {
-                  -moz-osx-font-smoothing: grayscale;
-                  -webkit-font-smoothing: antialiased;
                   color: #605e5c;
-                  font-family: 'Segoe UI', 'Segoe UI Web (West European)', 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif;
-                  font-size: 14px;
-                  font-weight: 400;
```

Comment:
I'm still confused why these values are disappearing. Can we leave in font-size/weight/family etc

Reply:
@micahgodbolt We were re-inheriting `theme.fonts.medium,` for each child. See [line](https://github.com/OfficeDev/office-ui-fabric-react/blob/e716eb14cba745960c0430249058d4a4ca400c66/packages/office-ui-fabric-react/src/components/TextField/TextField.styles.tsx#L260) for the parent style, and [this](https://github.com/OfficeDev/office-ui-fabric-react/blob/e716eb14cba745960c0430249058d4a4ca400c66/packages/office-ui-fabric-react/src/components/TextField/TextField.styles.tsx#L281) and [this](https://github.com/OfficeDev/office-ui-fabric-react/blob/e716eb14cba745960c0430249058d4a4ca400c66/packages/office-ui-fabric-react/src/components/TextField/TextField.styles.tsx#L288) for the inheritance.

I believe these are not necessary and removed these lines, which is the reason for the difference in the `.snap` files.

Reply:
@micahgodbolt if we want to bring these back, that's fine too.

Reply:
I'd rather bring them back. This is a small change , don't want the diff to b huge

Reply:
I have brought back the styles and updated the snapshots, so that the `font-weight`, `font-size` and `font-family` re-appear. https://github.com/OfficeDev/office-ui-fabric-react/pull/9488/commits/0df2a1647bfdf6d55535bb3f12aab9b9356011be

Reply:
they came back....but now it looks like they have been added. I just want to have the smallest code change possible, whatever that takes. We'll see how the screener tests come back.

Reply:
screener tests pass.

Reply:
@micahgodbolt, I have another commit https://github.com/OfficeDev/office-ui-fabric-react/pull/9488/commits/3eb0df5064609813e76b1572788a6645ee932066 where I change the order, and now just the newly added styles show.

PS: It was only the order of styles that was different, no styles were removed or newly added except for those for `Edge`.